smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/i18n/AMQPExceptions.java

```diff
@@ -0,0 +1,29 @@
+package io.smallrye.reactive.messaging.amqp.i18n;
+
+import org.jboss.logging.Messages;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageBundle;
+
+@MessageBundle(projectCode = "SRRML")
```

Comment:
What does the code mean? SmallRye Reactive Messaging L? Logging? Logger? Library?

Reply:
Maybe "SRMSG" instead?

Reply:
I am easy, just let me know what it should be.

Reply:
Making it SRMSG